### ISSUE TITLE
Face expression probabilities random on chrome android

### ISSUE THREAD

**Issue by davidstrahm** [justadudewhohacks/face-api.js#278]
After loading and detecting face from html video element, the probability distribution is correct for a few frames (happy 0.99, sad 0.0000343) etc.

After a few processed frames, the probabilities jump all over the place (see screenshots below).

The same code works fine when using it in chrome on my desktop machine (Nvidia GTX 1060)

Android 8.0.0
Chrome 73.0.3683.90
Samsung Galaxy S7

![probabilities](https://user-images.githubusercontent.com/16301487/56792792-b0ff6500-680a-11e9-9cdf-e6c362da217e.png)
```
const model = FaceApiFaceDetectorModel.TINY_FACE_DETECTOR;
const modelBaseUrl = '/assets/tfmodels';
const minConfidence = 0.5;
const inputSize = 256;
const scoreThreshold = 0.5;
const minFaceSize = 200;

await loadFaceDetectorModel(model, modelBaseUrl);
await faceapi.loadFaceExpressionModel(modelBaseUrl);

const options = getFaceDetectorOptions(model, minConfidence, inputSize, scoreThreshold, minFaceSize);

const result = await faceapi.detectSingleFace(video, options).withFaceExpressions();
```

**Comment by justadudewhohacks** [comment]
That's odd, especially since these are not even valid probabilities anymore. Looks like something is going wrong in `tf.softmax` here.

**Comment by davidstrahm** [comment]
Maybe this is related: https://github.com/tensorflow/tfjs/issues/1488, although face-api does not do batch predictions, does it? 

The probabilities are correct on newer phones. Do you have any plans to upgrade your tfjs dependency? Might be this is already fixed.

**Comment by justadudewhohacks** [comment]
The code that you posted does not do batch processing. Did you try to play around with enabling/disabling the specific features as @annxingyuan described in this issue? Does that help?

**Comment by davidstrahm** [comment]
Yes, I tried all 3 suggestions. The predictions are "better" when setting `tf.ENV.set('WEBGL_PACK', false)` (no more huge numbers), but still there are negative values and values in the range from -100 to 100.

**Comment by justadudewhohacks** [comment]
Ok. All I can do for now is upgrade tfjs-core to latest, which I am working on now.

**Comment by davidstrahm** [comment]
Thank you, I'll let you know in this issue if that helped

**Comment by davidstrahm** [comment]
Let me know if we can help

**Comment by davidstrahm** [comment]
Update: seems to work now on chrome mobile 74.0.3729.136
